src/configurations/destinations/af/ui-config.json

```diff
@@ -82,6 +82,24 @@
           "footerNote": "To send the custom properties to the root of eventValue.",
           "default": false
         },
+        {
+          "type": "dynamicCustomForm",
+          "label": "List of properties",
+          "value": "listOfProps",
+          "footerNote": "Add your name of properties that need to be sent at the root of eventValue.",
+          "customFields": [
+            {
+              "type": "textInput",
+              "value": "property",
+              "required": false,
+              "placeholder": "e.g: Price"
+            }
+          ],
+          "preRequisiteField": {
+            "name": "addPropertiesAtRoot",
+            "selectedValue": false
+          }
+        },
         {
           "type": "checkbox",
           "label": "Add af_currency to root of eventValue",
```

Comment:
Right now `afCurrencyAtRoot` is open only for one customer & once they start using `listOfProps`, afCurrencyAtRoot can be cleaned up cc: @ItsSudip